Update: [XSS Filter Evasion Cheat Sheet] #1254 (#1255)
OWASP/CheatSheetSeries: cheatsheets/XSS_Filter_Evasion_Cheat_Sheet.md

> The following is a "polygot test XSS payload." This test will execute in multiple contexts including html, script string, js and URL. Thank you to [Gareth Heyes](https://twitter.com/garethheyes) for this [contribution](https://twitter.com/garethheyes/status/997466212190781445).

Reviewer: ```suggestion
The following is a "polyglot test XSS payload." This test will execute in multiple contexts including html, script string, js and URL. (Based on this [tweet](https://twitter.com/garethheyes/status/997466212190781445) by [Gareth Heyes](https://twitter.com/garethheyes)).
```
